src/lib/openjp2/openjpeg.c

```diff
@@ -1049,7 +1093,12 @@ opj_stream_t* OPJ_CALLCONV opj_stream_create_file_stream(
 
     return l_stream;
 }
-
+
+OPJ_OFF_T opj_stream_skip_api(opj_stream_t * p_stream, OPJ_OFF_T p_size)

```

Comment:
this is unrelated to this pull request AFAICS. Better put that in another one

Reply:
Indeed, sorry! Some files (like [NITF](https://en.wikipedia.org/wiki/National_Imagery_Transmission_Format)) encapsulate JP2 in a file, so we need to offset the start of file reading. I did not find any way to do that so added this method.

Reply:
This is a bit self-promotion, but you are probably aware of the GDAL library (http://gdal.org/) that takes care of all this geo stuff already and deals with NITF.

Reply:
Yes, that´s an impressing and complete library! We do all our software in Java so it would require a lot of JNI bindings and we have implemented almost all image and sensor support we need already, this GML parsing being one of few missing parts.